### PR TITLE
features/RHPM_UR14 Crop and upload photo in update profile page

### PR DESCRIPTION
**Pull request by karthikkharvi**
features/RHPM_UR14 Crop and upload photo in update profile page

### CODE REVIEW

File: web/src/app/admin/users/user.component.ts [hunk 1]
```diff
@@ -142,18 +137,27 @@ export class UserComponent implements OnInit {
...
+                            } else {
+                                this.msgService.showError('Error');
+                            }
                         }
                     );
                 } else {
                     this.userServiceApp.updateUser(AppConstants.uriForUpdateUser, this.userModel, this.uploadedProfile).subscribe(
                         (data) => {
-                            this.showUsersList();
+                            debugger;
```
Comment (shikhaBasra) [inline web/src/app/admin/users/user.component.ts]: remove debugger statements

File: web/src/app/admin/users/user.component.ts [hunk 2]
```diff
@@ -142,18 +137,27 @@ export class UserComponent implements OnInit {
                 if (isNullOrUndefined(this.userModel.userId)) {
                     this.userServiceApp.addUser(AppConstants.uriForAddUser, this.userModel, this.uploadedProfile).subscribe(
                         (data) => {
```
Comment (shikhaBasra) [inline web/src/app/admin/users/user.component.ts:148]: handle exceptions inside error block

Comment (karthikkharvi) [reply web/src/app/admin/users/user.component.ts:148]: Done

File: web/src/app/admin/users/user.component.ts (hunk 1 above)
Comment (karthikkharvi) [reply web/src/app/admin/users/user.component.ts]: Done

**Review by shikhaBasra — APPROVED** [review]

**Pull request merged**